include/opendht/dht.h

```diff
@@ -292,8 +292,8 @@ class Dht {
        the target 8 turn out to be dead. */
     static constexpr unsigned SEARCH_NODES {14};
 
-    /* Concurrent requests during a search */
-    static constexpr unsigned SEARCH_REQUESTS {4};
+    /* Concurrent search nodes requested count */
```

Comment:
It is equivalent for now, but when the pagination behavior (#71) will be merged, the latter will be more explicit.